frappe/workflow/doctype/workflow_action/workflow_action.py

```diff
@@ -167,8 +263,10 @@ def get_next_possible_transitions(workflow_name, state, doc=None):
 	return transitions_to_return
 
 def get_users_next_action_data(transitions, doc):
+	roles = set()
```

Comment:
Not sure how I feel about changing the return type from `Dict` to `Tuple[Dict, set]` but as long as this is an internal function, nobody should care.